mteb/tasks/Classification/tur/TurkishMovieSentimentClassification.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class TurkishMovieSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TurkishMovieSentimentClassification",
+        description="Turkish Movie Review Dataset",
+        reference="https://www.win.tue.nl/~mpechen/publications/pubs/MT_WISDOM2013.pdf",
+        dataset={
+            "path": "asparius/Turkish-Movie-Review",
+            "revision": "409a4415cce5f6bcfca6d5f3ca3c408211ca00b3",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["tur-Latn"],
+        main_score="accuracy",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
```

Comment:
Required, if not specified then use "Not specified".

Reply:
There is a paragraph from the website of the datasets however there is no spesific mention of any licence. "We are working on making the software, source code and datasets created and used in this project available for the research community (as long as there are no NDA, IP ethical or proprietary concerns). Currently, the following datasets are available"

Reply:
Updated as "Not specified"